src/main/java/com/hms_networks/americas/sc/extensions/util/Base64.java

```diff
@@ -28,6 +28,9 @@
  * <p>Change Log:
  *
  * <ul>
+ *   <li>v2.3.7_J4 - Add support for Java 4 by removing all instances of <code>String.format()
+ *       </code>, <code>@Override</code>, and type parameters. (Author: HMS Networks, MU Americas
+ *       Solution Center)
```

Comment:
Should this update commit just remove this changelog list? We have not released any of the previous versions of this and none of our other classes have changelogs.

Reply:
We could remove it. I'm not really sure of any sort of conventions for this scenario. In general in this PR though, I tried to modify as little as possible to respect the developer who put it into the public domain, but also clearly indicate that we did modify it, and what we modified.

Reply:
I might look to @it-hms for a definitive answer on a decision like this, but I think leaving it and contributing our own changelog entry here allows us to clearly identify the changes we have made and retain the established format.

Reply:
I like it. It give us copyright protections.

Reply:
Sounds good!